HUD/N_0xf83d0febe75e62c9.md

```diff
@@ -5,20 +5,33 @@ ns: HUD
 
 ```c
 // 0xF83D0FEBE75E62C9
-void _0xF83D0FEBE75E62C9(Any p0, Any p1, Any p2, Any p3, Any p4, Any p5, Any p6, Any p7);
+void _0xF83D0FEBE75E62C9(cs_type(Any) Blip blip, Any p1, Any p2, cs_type(Any) float width, Any p4, cs_type(Any) float length, cs_type(Any) float heading, Any p7);
 ```
 
 ```
-NativeDB Added Parameter 9: Any p8
+NativeDB Added Parameter 9: int hudColour
 NativeDB Introduced: v1290
 ```
 
+Sets the 'data' for a cone for a blip
+
+See [here](https://docs.fivem.net/docs/game-references/hud-colors/) for the hud colours for the cone
+
+Name in scripts: `SETUP_FAKE_CONE_DATA()`
+
+For people who don't know how to convert degrees in radians and the other way around:
+
+```
+degrees = radians * 180 / pi
+radians = degrees * pi / 180
+```
+
 ## Parameters
-* **p0**:
-* **p1**:
-* **p2**:
-* **p3**:
-* **p4**:
-* **p5**:
-* **p6**:
-* **p7**:
+* **blip**: A blip from an entity, for example ch_prop_ch_cctv_cam_01a
```

Comment:
The example of a string (piece of text) is misguiding for type `Blip` (integer base type)

Reply:
> The example of a string (piece of text) is misguiding for type `Blip` (integer base type)

Nice catch I completely missed this, I tried to find the unknown `p*` args in the game code to see if this could be complemented a bit, but no joy.

Reply:
Ah I think I understand it now; we first need to get the blip id that's related to this "ch_prop_ch_cctv_cam_01a" object through some other means. If so, maybe explaining that process or leaving it out?

Reply:
> Ah I think I understand it now; we first need to get the blip id that's related to this "ch_prop_ch_cctv_cam_01a" object through some other means. If so, maybe explaining that process or leaving it out?

I think an easier way to explain it could be:
A blip from an entity, for example the player's entity or a vehicle entity, no? 😃

Reply:
The behavior on peds is weird. Thats why the ai blips exist for them. Rather exclude it than have a ‘native not working properly’, right?

Reply:
Would something like `GetBlipFromEntity(entity)` be used? Something like that would explain it pretty well. Can also make a small note that peds have undefined behavior.

Reply:
Is possible. Or `AddBlipForEntity()` (not a ped). But for peds the width, etc doesnt work sadly